cmake/Modules/Findlibdbi.cmake

```diff
@@ -0,0 +1,10 @@
+# Determine compiler flags for libdbi
+# Once done this will define
+# LIBDBI_FOUND - libdbi found
+
+find_path(LIBDBI_INCLUDE_DIR NAMES dbi.h HINTS "/usr/include/dbi")
+find_library(LIBDBI_DEV_LIBRARY NAMES dbi HINTS "/usr/lib/x86_64-linux-gnu")
+find_library(LIBDBI1_LIBRARY NAMES dbi HINTS "/usr/lib/x86_64-linux-gnu")
+find_library(LIBPGSQL_DRIVER_LIBRARY NAMES dbdpgsql HINTS "/usr/lib/x86_64-linux-gnu/dbd")
```

Comment:
These functions must have a variable LIBDBI_ROOT (similar to LIBCSV_DIR) which the user can set to point to the libdbi libraries and headers. This allows users to install the headers/libraries in non-standard locations and still compile GenomicsDB